generators/ci-cd/templates/jenkins/_jenkins.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  jenkins:
+    image: <%= DOCKER_JENKINS %>
+    ports:
+      - 49001:8080
+      - 50000:50000
+    #volumes:
+    # warning: be sure the local jenkins_home folder is created before using this volume
```

Comment:
Can we have commented lines to bind mount the docker sock and cli.
So you can test building in docker.